app/models/administrateur.rb

```diff
@@ -13,7 +13,6 @@ class Administrateur < ApplicationRecord
   has_many :dossiers, -> { state_not_brouillon }, through: :procedures
 
   before_validation -> { sanitize_email(:email) }
-  before_save :ensure_api_token
```

Comment:
Pinailliage : j'aurais extrait ça dans un commit à part, car ici d'une part on "do not generate one token by default", ce qui est fait dans ce fichier, et d'autre part, on "ensure a token is present for each API request"

Reply:
en fait, c'est discutable par ce qui si tu shipes l'un sans l'autre t'as un trou de sécu ... vu que certain admin non alors plus de token et que tu peux alors appeler l' (les :) ?) api sans token et recevoir de la data.

Reply:
Hmm je sais pas si je suis bien tout l'enchainement, je me serais dit que si on met l'un avant l'autre ça colmatait, mais je te fais confiance :)